Classes/Form/Field/Text.php

```diff
@@ -226,18 +225,10 @@ public function getRichtextConfiguration()
      */
     protected function getPageTsConfigForRichTextEditor()
     {
-        $configurationManager = $this->getObjectManager()->get(ConfigurationManagerInterface::class);
-        if ($configurationManager instanceof BackendConfigurationManager) {
-            $pageUid = $configurationManager->getCurrentPageId();
-        } else {
-            $root = $this->getRoot();
-            $pageUid = $root instanceof Form ? $root->getOption('record')['pid'] ?? 0 : 0;
-        }
+        $root = $this->getRoot();
+        $pageUid = $root instanceof Form ? $root->getOption('record')['pid'] ?? 0 : 0;
```

Comment:
`$root->getOption('record')` is NULL with flux 9.0.1, dumping `$root` gives me a Form-object with no `record` within the options path. So `$pageUid` gets always `0`, and the `RTE.default.preset` of the first page doesn't get loaded.